check-version.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+$majorVersionInUse = PHP_MAJOR_VERSION;
+$fullVersionInUse = phpversion();
+echo sprintf(
+    "Using PHP %s\n",
+    $fullVersionInUse
+);
+
+$url = 'https://www.php.net/releases/?json&version=' . $majorVersionInUse;
```

Comment:
I'd prefer an sprinf given that string and int aren't the same data type. But I understand not doing so.